java/admin/src/main/java/com/exadel/frs/service/ModelService.java

```diff
@@ -166,56 +169,44 @@ public Model cloneModel(
         return cloneAppModelAccessList;
     }
 
-    @Transactional
     public void cloneSubjects(final String sourceApiKey, final String newApiKey) {
```

Comment:
Will it work in one transaction? So if one of the subject's cloning fails, will it remove the already cloned subjects?